index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+        <header class="main-header"> 
+            <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
+            <nav class="nav-container">
+                <a href="#" class="menu"><img src="images/icon/menu.png"page logo=""></a>
```

Comment:
- [ ] For accessibility reasons and also it is among best practices to have an `alt` attribute to an `img` element so kindly add an `alt` attribute to the menu image 💪🏼, it can be `menu` if you prefer it.